README.md

```diff
@@ -54,3 +48,5 @@ git remote add nextjs-starter https://github.com/sdsna/nextjs-starter.git
 git fetch nextjs-starter
 git merge nextjs-starter/main
 ```
+
+Linting (and code formatting) also occurs automatically as a pre-commit hook thanks to [lint-staged](https://github.com/okonet/lint-staged) and [husky](https://github.com/typicode/husky).
```

Comment:
This is still broken compared to nextjs-starter: Missing `Linting` header plus first line. (not a major deal, but better to fix)